entities/entities/spawned_money/cl_init.lua

```diff
@@ -1,5 +1,8 @@
 include("shared.lua")
 
+local color_1 = Color(140, 0, 0, 100)
```

Comment:
Could you name the colors? They're dark red and white

Reply:
done

Reply:
Thanks!